sway-fmt-v2/src/lib.rs

```diff
@@ -6,7 +6,7 @@
 #![allow(dead_code)]
 pub mod config;
 mod constants;
-mod error;
+pub mod error;
```

Comment:
Is this needed to be public?

Reply:
got leftover from when I was fleshing out the change!